content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -102,25 +105,25 @@ build semver-based channels and upgrade graphs based solely on the versions of t
 no right or wrong answer for implementing this step as long as `opm validate` is successful.
 
 There are some guidelines to keep in mind though:
-- Once a bundle is present in an index, you should assume that one of your users has installed it. With that in mind,
+- Once a bundle is present in a Catalog, you should assume that one of your users has installed it. With that in mind,
   you should take care to avoid stranding users that have that version installed. Put another way, make sure that
-  all previously published bundles in an index have a path to the current/new channel head.
+  all previously published bundles in a Catalog have a path to the current/new channel head.
 - Keep the semantics of the upgrade edges you use in mind. `opm validate` is not able to tell you if you have a sane
   upgrade graph. To learn more about the upgrade graph of an operator, checkout the
   [creating an upgrade graph doc][upgrade-graph-doc].
 
-### Build and push the index image
+### Build and push the Catalog image
 
-Lastly, we can build and push our index:
+Lastly, we can build and push our Catalog image:
 
 ```sh
 $ docker build . \
-    -f example-operator-index.Dockerfile \
-    -t quay.io/example-inc/example-operator-index:latest
-$ docker push quay.io/example-inc/example-operator-index:latest
+    -f coolCatalog.Dockerfile \
```

Comment:
This is always the same name as the catalog directory when generated by `opm alpha generate dockerfile`
```suggestion
    -f cool-catalog.Dockerfile \
```